<launch>
    <node name="trajectory_controller" pkg="igvc_navigation" type="trajectory_controller" output="screen">
        <param name="topics/trajectory" value="trajectory"/>
    </node>
</launch>
